test/controls/control.test.js

```diff
@@ -51,3 +51,41 @@ test('parse no value', function (t) {
   t.notOk(c.value, null)
   t.end()
 })
+
+test('only value getter in sub-class with strict mode', function (t) {
+  // eslint-disable-next-line quotes
+  "use strict"
+
+  const assert = require('assert-plus')
```

Comment:
I'd rather not add any new code using this library. Additionally, we don't need a 100% complete control implementation for the test, just what is necessary to trigger the error.

Reply:
sure. no problem